release/release-it-release.json

```diff
@@ -6,9 +6,9 @@
     ],
     "after:bump": [
       "git add ../../CHANGELOG.md",
-      "echo $THE_KEY > resources/TELEMETRY_KEY",
+      "sed -i \"s/const TELEMETRY_KEY_ENCODED: string = TELEMETRY_DEFAULT_CONTENT/const TELEMETRY_KEY_ENCODED: string = '$THE_KEY'/\" src/services/reporter/TelemetryReporterImpl.ts",
```

Comment:
will this again break on macos?

Reply:
Release job runs only on `ubuntu-latest`.